tests/Custom/TestCase.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Behat\Mink\Tests\Driver\Custom;
+
+use PHPUnit\Framework\TestCase as BaseTestCase;
+use PHPUnit\Runner\Version;
+
+if (class_exists('PHPUnit\Runner\Version') && version_compare(Version::id(), '6.0.0', '>=')) {
+
+    /**
+     * Implementation for compatibility with PHPUnit 6+.
+     */
+    abstract class TestCase extends BaseTestCase
+    {
+    }
+
+} elseif (version_compare(\PHPUnit_Runner_Version::id(), '4.0.0', '>=')) {
+
+    /**
+     * Implementation for compatibility with PHPUnit 4 and 5.
+     */
+    abstract class TestCase extends \PHPUnit_Framework_TestCase
```

Comment:
No need, because PHPUnit 4.8.x (don't remember exact version) even though internally using `\PHPUnit_Framework_TestCase` still provides forward compatible namespaced `TestCase` class.

You just need to install correct version for PHP 5.5 and not use Travis CI provided one.

Reply:
How do you force the usage of a specific version of PHPUnit? I'm trying with this but it doesn't work (because it installs the local version of vendor/bin/phpunit: https://github.com/minkphp/MinkZombieDriver/pull/189/files#diff-354f30a63fb0907d4ad57269548329e3R38

Reply:
Using same IF you're using to install different PHPUnit version you can swap `phpunit` command later on for `vendor/bin/phpunit` command.

Reply:
The first version in which PHPUnit introduced the forward compatibility is 5.4.0, that's not compatible with PHP 5.5. Is there any other way to solve this, apart from the solution I implemented before (with a custom TestCase)?

Reply:
We have the same issue with PHP 5.3 (but not with PHP 5.4 🤔)

Reply:
> The first version in which PHPUnit introduced the forward compatibility is 5.4.0, that's not compatible with PHP 5.5. Is there any other way to solve this, apart from the solution I implemented before (with a custom TestCase)?

Nope. I'm sure was in 4.8.x versions as well. Please see: https://github.com/sebastianbergmann/phpunit/tree/4.8.36/src/ForwardCompatibility . Likely it was available in earlier 4.8.x version too.

Reply:
Ok, I got it. I was looking backwards from v5.5, and stopped to 5.4 but didn't check that v4 might have that forward compatibility. Doh! Anyway, first one is 4.35: https://github.com/sebastianbergmann/phpunit/blob/4.8.35/ChangeLog-4.8.md